axelor-stock/src/main/java/com/axelor/apps/stock/service/MassStockMoveServiceImpl.java

```diff
@@ -0,0 +1,130 @@
+package com.axelor.apps.stock.service;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.base.db.repo.SequenceRepository;
+import com.axelor.apps.base.db.repo.TraceBackRepository;
+import com.axelor.apps.base.service.administration.SequenceService;
+import com.axelor.apps.stock.db.MassStockMove;
+import com.axelor.apps.stock.db.PickedProducts;
+import com.axelor.apps.stock.db.StockLocationLine;
+import com.axelor.apps.stock.db.repo.MassStockMoveRepository;
+import com.axelor.apps.stock.db.repo.PickedProductsRepository;
+import com.axelor.apps.stock.db.repo.StockLocationLineRepository;
+import com.axelor.apps.stock.exception.StockExceptionMessage;
+import com.axelor.i18n.I18n;
+import com.axelor.inject.Beans;
+import com.google.inject.Inject;
+import com.google.inject.persist.Transactional;
+import java.math.BigDecimal;
+import java.util.List;
+import javax.swing.JOptionPane;
+
+public class MassStockMoveServiceImpl implements MassStockMoveService {
+
+  protected MassStockMoveRepository massStockMoveRepository;
+  protected SequenceService sequenceService;
+  protected PickedProductsRepository pickedProductRepo;
+
+  @Inject
+  public MassStockMoveServiceImpl(
+      MassStockMoveRepository massStockMoveRepository,
+      SequenceService sequenceService,
+      PickedProductsRepository pickedProductRepo) {
+    this.massStockMoveRepository = massStockMoveRepository;
+    this.sequenceService = sequenceService;
+    this.pickedProductRepo = pickedProductRepo;
+  }
+
+  @Override
+  @Transactional(rollbackOn = Exception.class)
+  public void importProductFromStockLocation(MassStockMove massStockMove) throws AxelorException {
+
+    try {
+
+      List<StockLocationLine> stockLocationLineList =
+          Beans.get(StockLocationLineRepository.class)
+              .all()
+              .filter("self.stockLocation = ?1", massStockMove.getCommonFromStockLocation())
+              .fetch();
+      List<StockLocationLine> detailsStockLocationLineList =
+          Beans.get(StockLocationLineRepository.class)
+              .all()
+              .filter("self.detailsStockLocation =?1", massStockMove.getCommonFromStockLocation())
+              .fetch();
+      if (stockLocationLineList.isEmpty() && detailsStockLocationLineList.isEmpty()) {
+        JOptionPane.showMessageDialog(
```

Comment:
For message dialog, response should be used.
This check should be done in controller